tests/router/common.py

```
@@ -622,98 +696,82 @@ def _test_router_overload_503(
-                # Start 50 long-running requests concurrently
+
+                async def send_and_record(req_id, payload):
+                    try:
+                        async with session.post(url, json=payload) as response:
+                            status = response.status
+                            if status == 200:
+                                # Consume streaming response to completion
+                                async for _ in response.content.iter_any():
+                                    pass
+                            return (req_id, status)
+                    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)